modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -470,6 +470,32 @@ private void postProcessEnumRefs(final Map<String, Object> models) {
                         var.isPrimitiveType = true;
                     }
                 }
+                for (CodegenProperty var : model.readWriteVars) {
```

Comment:
Previously this only flagged things in the allVars array which are not actually references to the objecsts in the readWrite or readOnly arrays, so it was missing out on this decoration